docs/submission/dictionary-overview.md

```diff
@@ -1,58 +1,55 @@
+- Field must be filled in the submitted TSV file.
+- When pared with the `Conditional` attribute, the field is only required if conditional requirements are met.
 
-#### Extended Fields
-![Extended](/assets/submission/dictionary-extended.svg) fields are not required for clinical data completion but it is strongly encouraged to provide as many extended fields as possible to help ensure data quality. In most cases, extended fields have a blank attribute, which means they are not required to be submitted. 
+![Core](/assets/submission/dictionary-core.svg) attribute indicates:
 
-If a field is classified as ![Extended](/assets/submission/dictionary-extended.svg) + ![ Conditional](/assets/submission/dictionary-conditional.svg), this means you can provide a value for this field only if the condition in the notes is met.
+- Field is part of the mandatory minimum set of clinical data that must be submitted.
+- The set of core clinical fields were defined by the [Tissue & Clinical Annotations Working Group](http://www.icgc-argo.org/page/84/tissue-clinical-annotation-working-group) which involved regular discussions with members of the working group and ARGO Programs.
+- Core clinical fields are commonly acquired in cohort-based studies and clinical trials and are required to address clinically relevant topics by cross entity analyses, and therefore constitute a critical element in the analysis of diverse ARGO Programs.
 
-### Field Type
-A field can be of type: Text, Integer, Number, or an Array of any of these types (array values are to be separated with a comma ","). An error will occur, such as *"The value is not permissible for this field"* if the incorrect field type is provided when uploading a file. 
+![Extended](/assets/submission/dictionary-extended.svg) attribute indicates:
 
-### Permissible Values
-Some fields will only accept certain values from a list that is provided in the permissible values column of the dictionary tables. It is mandatory that the value is written exactly as it is in the dictionary. Each value list is provided in alphabetical order and some are collapsed because they are quite long; please click the "# more" link to see the full list. 
-
-Other fields, such as IDs, are required to be written in a certain format. In this case, a regular expression is provided in this column. Some examples are also provided that link out to a regular expression resource that can be used to test if your value meets the regular expression. 
+- Field is not required for clinical data completion.
+- It is _strongly encouraged_ to provide as many extended fields as possible as collected by your research project to provide valuable data.
 
```

Comment:
- This field is not required for clinical data completion.

Again, the second bullet doesn't really follow the "indicates" well. Maybe it shouldn't be a bullet.

The sentence wording is a bit weird, maybe change to:
It is _strongly encouraged_ to provide as many extended fields as possible, to help ensure data quality for your research program.

Reply:
like not remove altogether right? I think it's still important to say they are strongly encouraged

Reply:
nope just the 2nd part- -left that.